components/tikv_util/src/config.rs

```diff
@@ -463,9 +475,14 @@ impl fmt::Display for ReadableDuration {
             write!(f, "{}s", dur / SECOND)?;
             dur %= SECOND;
         }
+        if dur >= MS {
+            written = true;
+            write!(f, "{}ms", dur / MS)?;
+            dur %= MS;
+        }
         if dur > 0 {
             written = true;
-            write!(f, "{}ms", dur)?;
+            write!(f, "{}us", dur)?;
```

Comment:
This seems a incompatible change. If a common configuration is set to `us` in the latest version and then rollback to older version, it will fail to start. I think it's acceptable, but better clarify in some places.

Reply:
Got it. I will update the doc. https://docs.pingcap.com/zh/tidb/stable/command-line-flags-for-tikv-configuration#tikv-%E9%85%8D%E7%BD%AE%E5%8F%82%E6%95%B0